src/models/classes/container.js

```diff
@@ -13,6 +13,10 @@ class Container extends StorageCore
         }
 
         super.addItem(product);
+
+        // Refresh warehouse
+        var warehouseController = new WarehouseController();
+        warehouseController._updateContainer();
```

Comment:
I feel this part should be done in 'simulation' controller, _i.e._ in a part of the code that does not exists at this time. Similarly, there is also no code yet for updating the model/view when a new order arrives.